spec.bs

```diff
@@ -726,7 +739,13 @@ following [=struct/items=]:
   :: a [=fencedframetype/fenced frame reporting map=]
 
   : <dfn>direct seller is seller</dfn>
-  :: a boolean
+  :: a [=boolean=]
```

Comment:
does this need an initial value, like the boolean below?

Reply:
I think it's always specified, but might as well give it a default.